packages/orchestrator-ui-components/src/components/WFOSubscription/utils/utils.tsx

```diff
@@ -63,3 +65,17 @@ export const getProductBlockTitle = (
         ? `${title.substring(0, MAX_LABEL_LENGTH)}...`
         : title;
 };
+
+export const flattenArrayProps = (
```

Comment:
The "flatten" part of the name makes me think of flatMap, it confused me a bit. I would say it is stil some kind op mapper. How about `mapSubscriptionActionToTranslationValues`?

Reply:
It doesn't map though, it flattens arrays in order to be able to be used as translation keys for this particular translations